lib/view_model/active_record/update_data.rb

```diff
@@ -24,45 +18,325 @@ def self.for_type(type)
       end
     end
 
-    class Append < self
+    class Append
       NAME = 'append'
       attr_accessor :before, :after
-      def self.schema
-        UpdateData::Schemas::APPEND_ACTION
+      attr_reader :contents
+      def initialize(contents)
+        @contents = contents
```

Comment:
They're used by both types of collections, so either ref strings or update datas.